docs/deploying.rst

```diff
@@ -0,0 +1,33 @@
+Deploying
+=========
+
+Make sure you are the only one deploying
+----------------------------------------
+
+There should be a way to make sure that only one person is trying to change a server at a time.
+
+Currently we use a wiki page that only Open Data Services Coop have access to. This means only Open Data Services Coop should deploy to live servers.
+
+TODO - Work out how we're going to make a deploy token work between Open Data Services and OCP, so that OCP staff can deploy. Document.
```

Comment:
Shall we JFDI and make a Owl of OCDS Deploy in the OCDS CRM, then document it here? That feels like a reasonable first attempt at a token?

Reply:
No, because at least one of our staff members does not have access to that CRM. Lets take a little time to think about this.